### PR TITLE
Improve footer

### PR DESCRIPTION
**Pull request by chlorenz**

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -1,14 +1,52 @@
-<footer>
-	<div class="row text-center">
+<div class="background-dark">
+	<div class="container">
```
Comment (MakisH) [inline _includes/footer.html:2]: I think we need more margin at the bottom. It currently seems to be quite tight at the bottom:

Screenshot from 2024-05-13 13-10-52

File: _includes/footer.html [hunk 2]
```diff
@@ -1,14 +1,52 @@
...
 
-		<a class="no-icon" rel="license" href="https://creativecommons.org/licenses/by/4.0/">
-			<img alt="Creative Commons License" src="images/licensebutton-by-40_88x31.png" />
-		</a>
-		<br />
-		This work is licensed under a <a rel="license" href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
+			<div class="col-md-4 col-xs-4 text-left">
+				<ul class="list-unstyled">
+					<li>
+	                    <a class="no-icon" href="https://precice.discourse.group"><!-- <i class="fab fa-discourse fa-lg"></i>  -->Forum on Discourse</a>
```
Comment (MakisH) [inline _includes/footer.html:25]: Any reason to keep the commented-out icons around?

File: _includes/footer.html [hunk 3]
```diff
@@ -1,14 +1,52 @@
...
+	                </li>
+	                <li>
+	                    <a class="no-icon" href="https://github.com/precice"><!-- <i class="fab fa-github fa-lg"></i>  -->preCICE on GitHub</a>
+	                </li>
+	                <li>
+						<a href="#">License</a>
```
Comment (MakisH) [inline _includes/footer.html:31]: On my list, I will open a PR asap.

File: _includes/footer.html [hunk 4]
```diff
@@ -1,14 +1,52 @@
...
+				</ul>
+			</div>
 
-	</div>
+			<div class="col-md-4 col-xs-4 text-right">
+				<a class="no-icon" rel="license" href="https://creativecommons.org/licenses/by/4.0/">
+					<img alt="Creative Commons License" src="images/licensebutton-by-40_88x31.png" />
+				</a>
+				<!-- <br />
+				This work is licensed under a <a rel="license" href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>. -->
```
Comment (MakisH) [inline _includes/footer.html:44]: This could still be the `alt` of the `img` above.

**Comment by fsimonis** [comment]
Replaced by #479

**Pull request closed without merging**